docs/concept/TAP7.1-solution-design.md

```diff
@@ -0,0 +1,425 @@
+# [TAP 7.1] Solution Design Hausanschluss Bundle Konzept
```

Comment:
@ds-jhartmann 
unique naming EDC "Hausanschluss" Bundle or Hausanschluss Bundle (in the whole doument)